### PR TITLE
Release of 4.28.1

### PR DESCRIPTION
**Pull request by trevor-vaughan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,16 @@ The headers used in [Keep a Changelog](http://keepachangelog.com) are:
 - Fixed - for any bug fixes.
 - Security - in case of vulnerabilities.
 
-# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.0...master)
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.1...master)
+
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.0...4.28.1) - 03-10-2021
```
Comment (bastelfreak) [inline CHANGELOG.md:25]: this should be like this?
```suggestion
# [4.28.1](https://github.com/puppetlabs/beaker/compare/4.28.0...4.28.1) - 03-10-2021
```

**Review by bastelfreak — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,7 +20,16 @@ The headers used in [Keep a Changelog](http://keepachangelog.com) are:
 - Fixed - for any bug fixes.
 - Security - in case of vulnerabilities.
 
-# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.0...master)
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.1...master)
```
Comment (ekohl) [inline CHANGELOG.md:23]: Please don't add an `Unreleased` entry when you're going to tag 4.28.1. Don't copy Puppetlabs' bad practices.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,7 +20,16 @@ The headers used in [Keep a Changelog](http://keepachangelog.com) are:
...
+
+# [4.28.1](https://github.com/puppetlabs/beaker/compare/4.28.0...4.28.1) - 03-10-2021
```
Comment (ekohl) [inline CHANGELOG.md:25]:
```suggestion
# [4.28.1](https://github.com/voxpupuli/beaker/compare/4.28.0...4.28.1) - 03-10-2021
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (trevor-vaughan) [reply CHANGELOG.md]: 🤷 I'm OK removing it.

**Review by bastelfreak — APPROVED** [review]

**Comment by trevor-vaughan** [comment]
@ekohl You OK with this now?

**Review by ekohl — APPROVED** [review]
I'm ok with this when the commits are squashed on merge.

**Review by bastelfreak — APPROVED** [review]

**Comment by trevor-vaughan** [comment]
:facepalm: I need to not do things in the morning. Sorry about the lack of squash.

**Pull request merged**